### DISCUSSION TITLE
ALTERNATIVE TO BACKGROUND MUSIC IN BIG SUR/MONTEREY

### DISCUSSION THREAD

**Discussion by ccgr33n** [kyleneideck/BackgroundMusic#569] [Q&A]
Has anyone found a good alternative to Background Music - which has been great. Looking to control volume and send to Blackhole (used to use Soundflower) to be able to record and have sound not be heard from computer or while recording, keeping all outside sound off recording.  Thank you.

**Comment by WARSAL** [comment]
I'm using now SoundControl, should work for your purposes. Con is fact that it's paid - 25$ (there's 14 days free trial) :D
